### PR TITLE
#162 plugin baseline and properly maintain api version

### PR DESCRIPTION
**Pull request by Emily-Jiang**
Signed-off-by: Emily Jiang <emijiang@uk.ibm.com>

### CODE REVIEW

File: api/src/main/java/org/eclipse/microprofile/health/package-info.java [hunk 1]
```diff
@@ -21,6 +21,6 @@
 /**
  * <p>Microprofile Health</p>
  **/
-@org.osgi.annotation.versioning.Version("1.0")
+@org.osgi.annotation.versioning.Version("1.1")
```
Comment (xstefank) [inline api/src/main/java/org/eclipse/microprofile/health/package-info.java]: I am not much experienced with OSGi but should the version be aligned with the release? So 2.0?

**Comment by xstefank** [comment]
@Emily-Jiang it seems that you must revalide your Eclipse agreement in order to pass Eclipse check :)

File: api/src/main/java/org/eclipse/microprofile/health/package-info.java (hunk 1 above)
Comment (antoinesd) [reply api/src/main/java/org/eclipse/microprofile/health/package-info.java]: I already asked @Emily-Jiang, apparently this doesn't have to follow spec version

Comment (Emily-Jiang) [reply api/src/main/java/org/eclipse/microprofile/health/package-info.java]: @antoinesd was right. The package version just maintain the status of api in that package. If nothing is changed in that package, the package version will stay the same even thought  it is a major release.

**Comment by Emily-Jiang** [comment]
I am not sure what happened. I renewed my eca recently and I was able to commit changes recently though.

**Comment by xstefank** [comment]
@Emily-Jiang Thanks for the explanation. For ECA problem you can try to force push your branch again with same sign commit to retrigger the check

**Pull request merged**